pkg/provider/provider.go

```diff
@@ -107,11 +111,26 @@ func (p *Provider) Validate(ctx context.Context, r *externaldata.ProviderRequest
 		dur := time.Since(start)
 		metrics.AttestationsRetrieved.Add(float64(len(bundles)))
 		metrics.AttestationsPullTimer.Observe(dur.Seconds())
-		log.Printf("validate: fetched %d OCI bundles in %s", len(bundles), dur)
+		slog.Info("validate: fetched OCI bundles",
+			"count", len(bundles),
+			"duration", dur.Seconds())
 
 		if err != nil {
 			metrics.AttestationsRetrieveFail.Inc()
-			log.Printf("validate: error fetching bundles: %s", err)
+			slog.Error("validate: error fetching bundles",
+				"image", key,
+				"error", err)
+			results = append(results, externaldata.Item{
+				Key:   key,
+				Error: key + "_unsigned",
```

Comment:
Nit: Do you think it would we worth omitting the `key` in the `error` field? 
I noticed that, when there's no attestation bundle, Gatekeeper logs
```
{
  "errors": [
    [
      "my-image:my-digest",
      "my-image:my-digest_unsigned"
    ]
  ],
  "responses": [],
  "status_code": 0,
  "system_error": ""
}
```
This was confusing me a bit at first, since I thought it might have done two validation attempts for the same image (one successful and one unsuccessful).